### PR TITLE
Rename CLI commands

### PR DESCRIPTION
**Pull request by clbartoli**
This PR renames some of the CLI commands so that the _get_ prefix disappear.
The old commands still working as an alias of the new ones.

The docs/ folder is deleted

### CODE REVIEW

File: docs/interface/json-rpc.md [hunk 1]
```diff
@@ -70,15 +70,15 @@ Response:
 {"jsonrpc":"2.0","result":true,"id":1}
 ```
 
-#### getBalance
+#### balance
```
Comment (tmpolaczyk) [inline docs/interface/json-rpc.md:73]: I think the documentation in `docs/` is deprecated

Comment (aesedepece) [reply docs/interface/json-rpc.md:73]: That's correct. We should simply remove that folder.

File: src/cli/node/with_node.rs [hunk 2]
```diff
@@ -248,7 +258,8 @@ pub enum Command {
         node: Option<SocketAddr>,
     },
     #[structopt(
-        name = "getUtxoInfo",
+        name = "utxoInfo",
+        alias = "getUtxoInfo",
```
Comment (aesedepece) [inline src/cli/node/with_node.rs:262]:
```suggestion
        name = "utxos"
        alias = "coins"
        alias = "utxoInfo",
        alias = "getUtxoInfo",
```

Comment (tmpolaczyk) [reply src/cli/node/with_node.rs:262]: `getCoins`?

**Pull request merged**